testinfra/plugin.py

```diff
@@ -117,6 +117,10 @@ def pytest_generate_tests(metafunc):
             ansible_inventory=metafunc.config.option.ansible_inventory,
             force_ansible=metafunc.config.option.force_ansible,
         )
+        if hasattr(metafunc.module, "testinfra_hosts_update"):
+            if not callable(metafunc.module.testinfra_update.callable):
```

Comment:
Shouldn't this be `callable(metafunc.module.testinfra_hosts_update)` ?

Reply:
I fixed bad offset from comment above, now it looks like this:
```python
        if hasattr(metafunc.module, "testinfra_hosts_update"):
            if not callable(metafunc.module.testinfra_update.callable):
                pytest.fail("testinfra_hosts_update must be a function")
            params = metafunc.module.testinfra_hosts_update(params)
```
The logic is that we complains if there is something called `testinfra_hosts_update` but it's not callable.